LoadTests/src/test/scala/default/RecordingSimulation.scala

```diff
@@ -569,27 +569,84 @@ class RecordingSimulation extends Simulation {
 			.check(
+				status.is(200),
+				regex("Change email address")))
+  		.pause(PAUSE_MIN_DUR, PAUSE_MAX_DUR)
+
+		val changeEmailAddress = exec(http("Change email address")
+			.post("/manage-account/change-email")
+			.headers(headers_3)
+			.formParam("__RequestVerificationToken", "${requestVerificationToken}")
+			.formParam("GovUk_Text_NewEmailAddress", "newemail@example.com")
+			.check(
+				status.is(200),
+				regex("We have sent a verification email to your new email address")))
+			.pause(PAUSE_MIN_DUR, PAUSE_MAX_DUR)
+
+		val visitChangePasswordPage = exec(http("Visit change password page")
+			.get("/manage-account/change-password-new")
+			.headers(headers_0)
+			.check(
+				status.is(200),
+				regex("Change your password")))
+  		.pause(PAUSE_MIN_DUR, PAUSE_MAX_DUR)
+
+		val changePassword = exec(http("Change password")
+			.post("/manage-account/change-password-new")
+			.headers(headers_3)
+			.formParam("__RequestVerificationToken", "${requestVerificationToken}")
+			.formParam("GovUk_Text_CurrentPassword", "Genderpaygap1")
+			.formParam("GovUk_Text_NewPassword", "Genderpaygap2")
+			.formParam("GovUk_Text_ConfirmNewPassword", "Genderpaygap2")
+			.check(
+				status.is(200),
+				regex("Your password has been changed successfully")))
+  		.pause(PAUSE_MIN_DUR, PAUSE_MAX_DUR)
+
+		val visitChangePersonalDetailsPage = exec(http("Visit change personal details page")
+			.get("/manage-account/change-personal-details")
+			.headers(headers_0)
+			.check(
+				status.is(200),
 				css("input[name='__RequestVerificationToken']", "value").saveAs("requestVerificationToken"),
 				regex("Change your personal details")))
-			.pause(PAUSE_MIN_DUR, PAUSE_MAX_DUR)
+  		.pause(PAUSE_MIN_DUR, PAUSE_MAX_DUR)
 
 		val changePersonalDetails = exec(http("Change personal details")
-			.post("/manage-account/change-details")
+			.post("/manage-account/change-personal-details")
 			.headers(headers_3)
-			.formParam("FirstName", "test user")
-			.formParam("LastName", "test user")
-			.formParam("JobTitle", "test job")
-			.formParam("ContactPhoneNumber", "")
-			.formParam("AllowContact", "true")
-			.formParam("SendUpdates", "false")
-			.formParam("AllowContact", "false")
 			.formParam("__RequestVerificationToken", "${requestVerificationToken}")
-				.check(regex("Your details have been updated successfully")))
+			.formParam("GovUk_Text_FirstName", "testing")
+			.formParam("GovUk_Text_LastName", "testing")
+			.formParam("GovUk_Text_JobTitle", "tester")
+			.formParam("GovUk_Text_ContactPhoneNumber", "")
+			.check(
+				status.is(200),
+				regex("Saved changes to personal details")))
+  		.pause(PAUSE_MIN_DUR, PAUSE_MAX_DUR)
+
+		val visitChangeContactPreferencesPage = exec(http("Visit change contact preferences page")
+			.get("/manage-account/change-contact-preferences")
+			.headers(headers_0)
+			.check(
+				status.is(200),
+				css("input[name='__RequestVerificationToken']", "value").saveAs("requestVerificationToken"),
```

Comment:
How come this changes?

Reply:
I don't think it does, but I wanted to make sure that `requestVerificationToken` has the right value wherever you're coming from on the site, so saved it here for safety!